driver-core/src/main/com/mongodb/operation/CommitTransactionOperation.java

```diff
@@ -104,4 +107,20 @@ static boolean shouldAddUnknownTransactionCommitResultLabel(final Throwable t) {
     protected String getCommandName() {
         return "commitTransaction";
     }
+
+    @Override
+    protected Function<BsonDocument, BsonDocument> getRetryCommandModifier() {
+        return new Function<BsonDocument, BsonDocument>() {
+            @Override
+            public BsonDocument apply(final BsonDocument command) {
+                BsonDocument retryCommand = command.clone();
```

Comment:
The clone is probably unnecessary.  The command document is created by our subclass, so we "own" it and are free to modify it.  I think...

Reply:
Changed.